src/file-format-verification/reducers/index.js

```diff
@@ -128,7 +128,7 @@ export const pagination = (state = defaultPagination, action) => {
     case END_PARSE:
       return {
         page: 1,
-        total: ((action.larErrors.length / ERRORS_PER_PAGE) >> 0) + 1,
+        total: Math.ceil(action.larErrors.length / ERRORS_PER_PAGE) || 1,
```

Comment:
Would it be better to use `defaultPagination.total` instead of `1` here?
(Same question for the `page` and `fade` values.)

Reply:
Created #160